docs/data/base/components/badge/badge.md

```diff
@@ -1,43 +1,149 @@
 ---
 product: base
-title: Unstyled React Badge component
+title: Unstyled React Badge component and hook
 components: BadgeUnstyled
 githubLabel: 'component: badge'
 packageName: '@mui/base'
 ---
 
 # Unstyled badge
 
-<p class="description">The `BadgeUnstyled` component generates a small label that is attached to its children elements.</p>
+<p class="description">The BadgeUnstyled component generates a small label that is attached to its child element.</p>
 
-```js
+## Introduction
+
+A badge is a small descriptor for UI elements.
+It typically sits on or near an element and indicates the status of that element by displaying a number, icon, or other short set of characters.
+
+The `BadgeUnstyled` component creates a badge that is applied to its child element.
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+## Component
+
+### Usage
+
+After [installation](/base/getting-started/installation/), you can start building with this component using the following basic elements:
+
+```jsx
 import BadgeUnstyled from '@mui/base/BadgeUnstyled';
+
+export default function MyApp() {
+  return (
+    <BadgeUnstyled>{/* the element that the badge is attached to */}</BadgeUnstyled>
+  );
+}
 ```
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+### Basics
```

Comment:
When landing on https://deploy-preview-33156--material-ui.netlify.app/base/react-badge you need to scroll a lot before you can find an actual demo of the component or one that you can copy and paste into your project. It contrasts with what we had before https://mui.com/base/react-badge/ or https://www.radix-ui.com/docs/primitives/components/tooltip. So I conclude that we miss an introduction demo, maybe under `### Basics`?

Reply:
This is reinforced by this feedback https://mui-org.slack.com/archives/C0170JAM7ML/p1659457895980239?thread_ts=1658732401.469249&cid=C0170JAM7ML

Reply:
That is a good point. We could definitely use a basic intro demo here, and probably for some other Base components as well.

Reply:
I will raise this discussion at the next team meeting, seeing

- https://reshaped.so/content/docs/components/switch
- https://headlessui.com/react/switch
- https://www.radix-ui.com/docs/primitives/components/switch

it feels nice to see a full demo first before everything else. It can even then turn into an SEO hack https://www.notion.so/mui-org/Page-image-generator-75bc177fefe64c0b899a42a6f919a907.